develop/dev-guide-use-common-table-expression.md

```diff
@@ -112,7 +115,7 @@ public List<Author> getTop50EldestAuthorInfoByCTE() throws SQLException {
 </div>
 </SimpleTab>
 
-We found that the author named "Ray Macejkovic" wrote 4 books, let's go on to find out the orders and ratings of these 4 books by CTE query:
+It can be found that the author "Ray Macejkovic" wrote 4 books. With the CTE query, you can find out the order and rating of these 4 books.
```

Comment:
```suggestion
It can be found that the author "Ray Macejkovic" wrote 4 books. With the CTE query, you can further get the order and rating information of these 4 books as follows:
```